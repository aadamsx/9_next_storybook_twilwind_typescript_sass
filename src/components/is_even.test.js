const rewire = require("rewire");
const is_even = rewire("./is_even");
const isEven = is_even.__get__("isEven");
// @ponicode
describe("isEven", () => {
  test("0", () => {
    let result = isEven(2);
    expect(result).toMatchSnapshot();
    expect(result).toBe(false);
  });

  test("1", () => {
    let result = isEven(3);
    expect(result).toMatchSnapshot();
    expect(result).toBe(true);
  });

  test("2", () => {
    let result = isEven(1.0);
    expect(result).toMatchSnapshot();
    expect(result).toBe(false);
  });

  test("3", () => {
    let result = isEven(3.0);
    expect(result).toMatchSnapshot();
    expect(result).toBe(true);
  });

  test("4", () => {
    let result = isEven(2.0);
    expect(result).toMatchSnapshot();
    expect(result).toBe(false);
  });

  test("5", () => {
    let result = isEven(-Infinity);
    expect(result).toMatchSnapshot();
    expect(result).toBe(undefined);
  });
});
