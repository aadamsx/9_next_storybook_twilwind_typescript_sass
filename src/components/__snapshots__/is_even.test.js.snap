// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`isEven 0 1`] = `true`;

exports[`isEven 1 1`] = `false`;

exports[`isEven 2 1`] = `false`;

exports[`isEven 3 1`] = `false`;

exports[`isEven 4 1`] = `true`;

exports[`isEven 5 1`] = `false`;
